lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -638,6 +638,17 @@ def test_course_image(self):
         self.assertTrue(url.startswith('/static/toy_course_dir/'))
         self.course.static_asset_path = ""
 
+    @ddt.data(ModuleStoreEnum.Type.split)
```

Comment:
there's no reason to use `ddt` if you're only putting in one modulestore kind here! Please add the mongo modulestore here too so we make sure they have the same behavior